zingolib/src/wallet/describe.rs

```diff
@@ -164,15 +132,39 @@ impl LightWallet {
             }),
             Box::new(|nnmd, _| !nnmd.pending_receipt()),
         ];
-        self.shielded_balance::<D>(target_addr, filters).await
+        self.shielded_balance::<D>(filters).await
+    }
+    /// The amount in orchard notes, not yet on chain
+    pub async fn pending_balance<D: DomainWalletExt>(&self) -> Option<u64>
+    where
+        <D as Domain>::Recipient: Recipient,
+        <D as Domain>::Note: PartialEq + Clone,
+    {
+        self.shielded_balance::<D>(&[Box::new(|note, _| note.pending_receipt())])
```

Comment:
this should be based of confirmation status, the pending reciept is based on nullifier calculation and probably less stable
```suggestion
        self.shielded_balance::<D>(&[Box::new(|_, transaction| transaction.status.is_pending())])
```